filters/ignore_new_executors.hpp

```diff
@@ -0,0 +1,56 @@
+#include <ctime>
+#include <memory>
+
+#include "messages/serenity.hpp"
+
+#include "serenity/executor_map.hpp"
+#include "serenity/serenity.hpp"
+
+#include "stout/option.hpp"
+
+#ifndef SERENITY_IGNORE_NEW_TASKS_HPP
```

Comment:
And it should be SERENITY_IGNORE_NEW_EXECUTORS_FILTER_HPP to be consistent with class name.